README.md

```diff
@@ -185,32 +186,51 @@ required to perform this process.
    }
    ``` 
 
-1. In DCP infra run the script `setup_fusillade.py {stage} --file {group}.json`
+1. In DCP infra run the script `setup_fusillade.py {stage} --file groups.json`
    to add your group to Fusillade. If any roles specified in the group do not
    exist, they will not be added to the group. If any user specified in the group
    are not in the group, they will be added to the group.
 
 ### As a DCP operator how do I remove a role from a group?
+#### Option 1: Using `setup_fusillade.py`
+1. Remove the role from the group defined in `dcp-fusillade/config/groups.json`. 
+1. Run the script `setup_fusillade.py {stage} --file groups.json`. Alternatively if using CICD, make a pull request 
+   with the changes and allow your CICD flow to apply the changes.
 
-Use the `PUT /v1/group/{group_id}/roles` endpoint to remove a role from a group.
-
-This must be done using a user account or service account that has permission to modify groups.
+#### Option 2: Using API
+1. Use the `PUT /v1/group/{group_id}/roles` endpoint to remove a role from a group. This must be done using a user 
```

Comment:
May need to clarify this is a three-step process: (1) get the existing list of roles, (2) remove the specified role, and (3) use the PUT endpoint to push the new list of roles associated with this group to the Fusillade server